srcpkgs/lcalc/template

```diff
@@ -0,0 +1,45 @@
+# Template file for 'lcalc'
+pkgname=lcalc
+version=1.23
+revision=1
+build_wrksrc="src"
+build_style="gnu-makefile"
+makedepends="pari-devel"
+short_desc="Michael Rubinstein's L-function calculator"
+maintainer="Gonzalo Tornaría <tornaria@cmat.edu.uy>"
+license="GPL-2.0-or-later"
+homepage="https://mirrors.mit.edu/sage/spkg/upstream/lcalc/"
+distfiles="https://mirrors.mit.edu/sage/spkg/upstream/lcalc/lcalc-${version}.tar.bz2"
+checksum=83d4253ec2f38553b21190d6d0c6b71bab7ea14717f6dde5bd18f60775d3cdb0
+patch_args="-Np1"
+nocross=yes
+
+pre_build() {
+	CC="$CXX"
+	LDFLAGS+=" -lpari"
+	make_build_args+=" PARI_DEFINE=-DINCLUDE_PARI"
+}
+
+pre_install() {
+	mkdir -p ${DESTDIR}/usr/{bin,lib,include}
+	make_install_args+=" INSTALL_DIR=${DESTDIR}/usr"
+}
+
+do_check() {
+	echo "Testing lcalc ..."
+	. "${FILESDIR}/test" > test.log
+	case "$XBPS_TARGET_MACHINE" in
+		i686) 	OUT="${FILESDIR}/test.i686" ;;
```

Comment:
I don't like hardcoding per architecture, but also don't see easy way to calculate relative error without perl or such.

Reply:
Re. my comment below, I'd rather have perl in `hostmakedepends` than the vendored test results (which would also break if the example program changes anything, right?).